src/main/kotlin/io/zerobase/smarttracing/GraphDao.kt

```diff
@@ -53,4 +57,183 @@ class GraphDao(private val driver: Driver) {
             }
         }
     }
+
+    /**
+     * Creates an organization node inside neo4j.
+     *
+     * @param organization name of the organization that is being created.
+     * @param phone phone number for the contact for the organization.
+     * @param email email for the contact for the organization.
+     * @param contactName name of the contact for the organization.
+     * @param address address of the organization.
+     * @param testing if the organization owns sites that can do testing.
+     * @param multiSite reporting if the organization has multiple sites.
+     *
+     * @return organization id.
+     */
+    fun createOrganization(organization: String, phone: String, email: String,
+                           contactName: String, address: String, testing: Boolean,
+                           multiSite: Boolean): OrganizationId? {
+        val validNum = phoneUtil.isValidNumber(phoneUtil.parseAndKeepRawInput(phone, "US"))
+
+        if (!validNum) {
+            return null
+        }
+
+        return driver.session().use {
+            it.writeTransaction { txn ->
+                val result = txn.run(
+                        """
+                        CREATE (o:Organization {
+                                    id: '${UUID.randomUUID()}',
+                                    phone: '${phone}',
+                                    email: '${email}',
+                                    contactname: '${contactName}',
+                                    organization: '${organization}',
+                                    address: '${address}',
+                                    verified: 'false',
+                                    testing: '${testing}',
```

Comment:
naming this field just "testing" is going to make it harder to understand for someone who hasn't been in all the model discussions. rename to "hasTestingFacilities" for clarity